### PR TITLE
Aggregation of drive-by cosmetic changes for trait-upcasting PR

### PR DESCRIPTION
**Pull request by alexreg**
Cherry-picked from #60900.

As requested by @Centril (and @nikomatsakis, I believe).

r? @Centril

### CODE REVIEW

File: src/librustc_codegen_llvm/debuginfo/metadata.rs [hunk 1]
```diff
@@ -1030,7 +1028,7 @@ impl MetadataCreationResult<'ll> {
 }
 
 // Description of a type member, which can either be a regular field (as in
```
Comment (Centril) [inline src/librustc_codegen_llvm/debuginfo/metadata.rs]:
```suggestion
/// Description of a type member, which can either be a regular field (as in
```

File: src/librustc_mir/monomorphize/collector.rs [hunk 2]
```diff
@@ -769,8 +767,8 @@ fn visit_instance_use<'tcx>(
     }
 }
 
-// Returns true if we should codegen an instance in the local crate.
-// Returns false if we can just link to the upstream crate and therefore don't
+// Returns `true` if we should codegen an instance in the local crate.
+// Returns `false` if we can just link to the upstream crate and therefore don't
 // need a mono item.
 fn should_monomorphize_locally<'tcx>(tcx: TyCtxt<'tcx>, instance: &Instance<'tcx>) -> bool {
```
Comment (Centril) [inline src/librustc_mir/monomorphize/collector.rs]: doc comment?

**Comment by Centril** [comment]
Thanks; r=me with changes ^--- applied.

File: src/librustc_codegen_llvm/debuginfo/metadata.rs [hunk 3]
```diff
@@ -70,7 +70,7 @@ impl fmt::Debug for llvm::Metadata {
 }
 
 // From DWARF 5.
-// See http://www.dwarfstd.org/ShowIssue.php?issue=140129.1
+// See http://www.dwarfstd.org/ShowIssue.php?issue=140129.1.
```
[the remark this answers is not in the document]
Comment (alexreg) [reply src/librustc_codegen_llvm/debuginfo/metadata.rs]: It applies to all the comments below, so that's kind of hard...

**Comment by Centril** [comment]
Fairly bit-rotty; @bors p=1

**Pull request merged**